models.py

```diff
@@ -2,27 +2,32 @@
 from statsmodels.tsa.arima_model import ARIMA
 import warnings
 import numpy
-def status_quo_model(X):
+
+def status_quo_model(X,data):
```

Comment:
This changes the interface of `status_quo_model`, which will cause syntax errors in the files that call `status_quo_model`. I think there are two options here:

1) Change the code at all the places that call `status_quo_model`
2) (IMO the better option) make the second argument optional, so that the code behaves the same way as before when called without the new argument. Document the expected behavior.

Reply:
Actually, a third and possibly better option: we want the models have a consistent interface so that they can be easily swapped out for one another elsewhere in the code. Since we're already seeing that each model will require different preprocessing, it might just be a good idea to pass `data` and nothing else into each model. Then, each model will call the preprocessing functions it needs from `data`. So you can remove `X` as an argument and just use `data` as the only arg for all models.

We alternatively could have them inherit an interface from an abstract superclass but that seems unnecessary.